src/ucl_util.c

```diff
@@ -1131,15 +1154,48 @@ ucl_include_common (const unsigned char *data, size_t len,
 		}
 	}
 
-	if (*data == '/' || *data == '.') {
-		/* Try to load a file */
-		return ucl_include_file (data, len, parser, need_sign, !try_load,
-				allow_glob, priority);
+	if (parser->includepaths == NULL) {
+		if (allow_url && strnstr(data, "://", len) != NULL) {
```

Comment:
`strnstr` is not standard function (it's FreeBSD specific), and hence it should not be used.